### ISSUE TITLE
Witch doesn't extinguish in water

### ISSUE THREAD

**Issue by alexiscoutinho** [Tsuey/L4D2-Community-Update#177]
The Witch doesn't stop burning when she goes in water (i.e. c2m1 "riverbed") like the Tank does.

**Comment by FurtadoPires** [comment]
This one will take a lot of people by surprise if it's fixed

**Comment by alexiscoutinho** [comment]
Yeah, but a lot of things in TLS update annoyed players and took people by surprise.
